lib/Fhp/Options/Credentials.php

```diff
@@ -8,9 +8,9 @@
 class Credentials
 {
     /** @var string */
-    public $benutzerkennung;
+    private $benutzerkennung;
     /** @var string */
-    public $pin;
+    private $pin;
```

Comment:
Ich wäre für protected, ich musste schon so oft bei GitHub Projekten PRs machen nur um private in protected zu ändern.

Reply:
Ok, erledigt. `json_encode()` nimmt keine protected Felder, also passt das.